src/preprocess/dataset_split/audio_data_balancer.py

```diff
@@ -0,0 +1,174 @@
+import os
+import random
+import shutil
+
+import librosa
+import numpy as np
+import pandas as pd
+import soundfile as sf
+
+from src.preprocess.audio_waveform import preprocess
+
+
+random.seed(42)
+
+
+def _load_waveform(file_path: str) -> np.ndarray:
+    y_waveform, sampling_rate_waveform = librosa.load(file_path)
```

Comment:
```suggestion
    y_waveform, sampling_rate_waveform = librosa.load(file_path, sr=32_000)
```
Depending on which version we have there is otherwise a default

Reply:
I do not completely understand why we should have a default? I think librosa detects the sampling rate and it is for the data at hand 22050, I think